ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -874,23 +874,21 @@ const triggerSet: TriggerSet<Data> = {
         const defamation = data.defamationColor;
         if (defamation === undefined)
           return;
-        if (regression === 'remote') {
-          const color = defamation === 'red' ? output['blue']!() : output['red']!();
-          return output.nearTether!({ color: color });
-        }
+        const color = defamation === 'red' ? output.blue!() : output.red!();
+        if (regression === 'remote')
+          return output.farTether!({ color: color });
 
         if (parseFloat(matches.duration) < 80)
-          return output.farTether!({ color: output[defamation]!() });
+          return output.nearTether!({ color: color });
 
-        const color = defamation === 'red' ? output['blue']!() : output['red']!();
         return output.finalTowerFar!({ color: color });
       },
       outputStrings: {
-        nearTether: {
+        farTether: {
           en: 'Stack by ${color} Tower',
         },
-        farTether: {
-          en: 'Get ${color} Defamation',
+        nearTether: {
+          en: 'Outside ${color} Towers',
```

Comment:
"Get" here felt too much like "get rot", expecting you to move and go pick up defamation rather than stand outside.  So, I tried to make this more consistent with the "stack" trigger wording avove.

Reply:
Works for me.